oxidation/bindings/android/libparsec/rust/src/lib.rs

```diff
@@ -23,14 +23,24 @@ pub extern "C" fn Java_com_scille_libparsec_Runtime_startRuntime(
     true
 }
 
+/// Interface between NodeJs, Rust and Android
```

Comment:
There no NodeJS here: the bridge is between web (JS) and native (Rust) through Java (Android)